frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -87,6 +91,26 @@ export type Props = {
   runtimeData?: RuntimeData;
 };
 
+type CtaProps = {
+  link?: string;
+  label: string;
+  subscribed?: boolean;
+};
+
+const CtaLinkButton = (props: CtaProps) => {
+  const hasSubscription = props.subscribed;
+
+  return (
+    <>
+      {!hasSubscription ? (
+        <Link href="/premium#pricing">
+          <span className={styles.cta}>{props.label}</span>
+        </Link>
+      ) : null}
```

Comment:
Change the link to `/premium#pricing` as opposed to straight to Subplat. I left the `link` prop as an option we may use in the future.